pkg/cmd/roachtest/tests/backup_restore_roundtrip.go

```diff
@@ -372,3 +354,65 @@ func workloadWithCancel(m cluster.Monitor, fn func(ctx context.Context) error) f
 	})
 	return cancelWorkload
 }
+
+// setupBackupRestoreTestUtils sets up a CommonTestUtils instance for backup and
+// restore tests and initializes some useful settings.
+func setupBackupRestoreTestUtils(
+	ctx context.Context,
+	t test.Test,
+	c cluster.Cluster,
+	m cluster.Monitor,
+	rng *rand.Rand,
+	testOpts ...commonTestOption,
+) (*CommonTestUtils, error) {
+	connectFunc := func(node int) (*gosql.DB, error) {
+		conn, err := c.ConnE(ctx, t.L(), node)
+		if err != nil {
+			return nil, fmt.Errorf("failed to connect to node %d: %w", node, err)
+		}
+
+		return conn, err
+	}
+	// TODO (msbutler): enable compaction for online restore test once inc layer limit is increased.
+	testUtils, err := newCommonTestUtils(ctx, t, c, connectFunc, c.CRDBNodes(), testOpts...)
+	if err != nil {
+		return nil, err
+	}
+	if err := testUtils.setShortJobIntervals(ctx, rng); err != nil {
```

Comment:
it feels a bit weird to set cluster settings in this test utils setup function, but i'm fine to keep as is.